doc/classes/ItemList.xml

```diff
@@ -73,6 +73,7 @@
 			<argument index="0" name="idx" type="int">
 			</argument>
 			<description>
+				Returns the [Color] set by [member set_custom_bg_color]. Default value is [code]Color(0,0,0,0)[/code].
```

Comment:
We try to follow the GDScript code style from the docs in the code examples, so this should be `Color(0, 0, 0, 0)`: https://docs.godotengine.org/en/3.1/getting_started/scripting/gdscript/gdscript_styleguide.html#whitespace